vagrant/boxes/Vagrantfile_archlinux_systemd

```diff
@@ -34,7 +34,7 @@ Vagrant.configure("2") do |config|
     pacman --needed --noconfirm -S base base-devel bpf acl audit bash-completion clang compiler-rt docbook-xsl ethtool \
         git gnu-efi-libs gperf intltool iptables kexec-tools kmod libcap libelf libfido2 libgcrypt libidn2 \
         libmicrohttpd libpwquality libseccomp libutil-linux libxkbcommon libxslt linux-api-headers llvm llvm-libs lz4 meson ninja \
-        p11-kit pam pcre2 python-lxml qrencode quota-tools rust tpm2-pkcs11 xz
+        p11-kit pam pcre2 python-jinja2 python-lxml qrencode quota-tools rust tpm2-pkcs11 xz
```

Comment:
Looks like the Arch package is just `python-jinja' https://archlinux.org/packages/community/any/python-jinja/